### ISSUE TITLE
Sending Authentication/token Info

### ISSUE THREAD

**Issue by vaibhav15s** [westy92/html-pdf-chrome#142]
Hi
Is there a way I can send token information or basic authentication?

**Comment by olecom** [comment]
> basic authentication

@vaibhav15s I guess you can take needed page directly via `'chrome-remote-interface'`

https://github.com/westy92/html-pdf-chrome/blob/2fbe33f291a28cda75ce45c79d5c2f88b724a6fd/src/index.ts#L100-L110

Try to place https://chromedevtools.github.io/devtools-protocol/tot/Network#method-setExtraHTTPHeaders
somewhere after `client.Network.enable(),` in example below:
```js
async function htmlTune(html){
    // get HTML into Chrome to get computed style via `runtimeConsoleHandler`
    var options = {
        port: 9222,
        runtimeConsoleHandler: getTemplateImageDimentions
    }
    // https://chromedevtools.github.io/devtools-protocol/tot/
    var tab = await CDP.New(options)
    var client = await CDP(Object.assign({}, options, { target: tab }))

    // Enable events to be used here
    await Promise.all([
        client.Network.clearBrowserCache(),
        client.Network.enable(),
        client.Page.enable(),
        client.Runtime.enable(),
    ]);
    if (options.runtimeConsoleHandler) {
        client.Runtime.consoleAPICalled(options.runtimeConsoleHandler);
    }

    const { frameTree } = await client.Page.getResourceTree();
    // open HTML and get info via `runtimeConsoleHandler`
    await client.Page.setDocumentContent({ html, frameId: frameTree.frame.id }),
    await client.Page.loadEventFired()
    await CDP.Close(Object.assign(options, { id: tab.id }));
}
```

**Comment by westy92** [comment]
I just published [v0.6.0](https://github.com/westy92/html-pdf-chrome/releases/tag/v0.6.0) which now supports sending custom HTTP headers with a request!
https://github.com/westy92/html-pdf-chrome#http-headers
